src/wazuh_modules/vulnerability_scanner/tests/unit/versionMatcher_test.cpp

```diff
@@ -700,8 +700,8 @@ TEST_F(VersionMatcherTest, compareRpmVer_rmpPackages)
                                       "1.0.1-10.module_el8.5.0+150+5f0dbea0.alma.ppc64le",
                                       VersionObjectType::RPM),
               VersionComparisonResult::A_EQUAL_B); // Alma
-    EXPECT_EQ(VersionMatcher::compare("2.2.20-2.el8.centos", "2.2.20-2.el8", VersionObjectType::RPM),
-              VersionComparisonResult::A_EQUAL_B); // CentOS
+    EXPECT_EQ(VersionMatcher::compare("2.2.20", "2.2.20-2.el8", VersionObjectType::RPM),
+              VersionComparisonResult::A_LESS_THAN_B); // No release
```

Comment:
Lets keep this test with the expected behaviour

```suggestion
    EXPECT_EQ(VersionMatcher::compare("2.2.20", "2.2.20-2.el8", VersionObjectType::RPM),
              VersionComparisonResult::A_LESS_THAN_B); // No release
    EXPECT_EQ(VersionMatcher::compare("2.2.20-2.el8.centos", "2.2.20-2.el8", VersionObjectType::RPM),
              VersionComparisonResult::A_GREATER_THAN_B); // CentOS
```

Reply:
`2.2.20-2.el8.centos` is a little probable version, the correct way should be `2.2.20-2.centos`. We should consider? I think is covered by another test